pkgs/applications/misc/st/xst.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchurl, pkgconfig, libX11, ncurses, libXext, libXft , fontconfig
+, patches ? [], extraLibs ? []}:
+
+with stdenv.lib;
+
+let
+  version = "0.7.1";
+  name = "xst-${version}";
+  patches' = if patches == null then [] else patches;
```

Comment:
Thanks for the examples! And yes, I agree that's much simpler. I think they just wanted a slightly simpler interface to customize st from configuration.nix. Updated PR.